src/pages/public_registration/PublicPublicationContext.tsx

```diff
@@ -144,6 +144,11 @@ export const PublicPublishedInContent = ({ id }: { id: string | null }) => {
   ) : null;
 };
 
+export const RevisionInformation = ({ revision }: { revision?: Revision | undefined | null }) => {
+  const { t } = useTranslation();
+  return Revision.REVISED === revision ? <Typography>{t('registration.is_revision')}</Typography> : <></>;
+};
+
```

Comment:
Trenger vel ikke å ha `undefined` og `null` når du uansett bruker en ternary her. Den kan forsåvidt også byttes ut for å komprimere litt.
```suggestion
export const RevisionInformation = ({ revision }: { revision: Revision }) => {
  const { t } = useTranslation();
  return revision === Revision.REVISED && <Typography>{t('registration.is_revision')}</Typography>;
};
```

Reply:
VI legger som regel props i en egen interface også, men det er vel ikke megaviktig

Reply:
Får feilmelding når jeg kompilerer uten undefined | null:

`TS2322: Type 'Revision | null | undefined' is not assignable to type 'Revision'.
  Type 'undefined' is not assignable to type 'Revision'.
    124 |                 isbnList={(registration as BookRegistration).entityDescription.reference?.publicationContext.isbnList}
    125 |               />
  > 126 |               <RevisionInformation revision={(publicationContext as BookPublicationContext).revision} />
        |                                    ^^^^^^^^
    127 |             </>
    128 |           ) : isDegree(publicationInstance.type) ? (
    129 |             <>


`

Reply:
Parameter bør være identisk med det som er i datamodellen, altså: `revision?: Revision | null`

Reply:
og enten `&&` som Isar foreslår, eller returnere `null` i stedet for `<></>`. Det vil gå for det samme i praksis

Reply:
Jeg får compile error på alt dere foreslår her. Så gjerne check ut branchen min og commit direkte der.